guides/CHANGELOG.md

```diff
@@ -18,4 +18,8 @@
 
     *Andrey Nering*
 
+*   Updated Unicode version to 8.0.0
+
+    *Anshul Sharma*
```

Comment:
Oh, I forgot to mention, but this should be in the activesupport changelog instead (there is where the change happened, right?). Also you can put the entry on the top :)